[ADD] account_payment_group_currency

jjscarafia opened the pull request.

jjscarafia committed:
[ADD] account_payment_group_currency

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/570).

jjscarafia committed:
[FIX] ugly fix for bypassing odoo constraint

Por ahora pisamos metodo para poder entregar prueba de concepto. Lo ideal sería parchearlo de otra manera

jjscarafia commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.
